README.rst

```diff
@@ -70,9 +70,7 @@ Here is a table of the files to translate and their styles. Everything
 that is not mentioned here, should be asked about if in doubt. Not all
 pages make sense to translate.
 
-.. note::
-
-   important
+.. important::
```

Comment:
Cool, I am using these ones far too little. I think docutils hates my `admonition` usages these days, I should strive to replace them. Also interesting what this does for Sphinx rendering.